src/device/ai/ai_controller.c

```diff
@@ -139,12 +140,14 @@ void init_ai(struct ai_controller* ai,
              struct r4300_core* r4300,
              struct ri_controller* ri,
              struct vi_controller* vi,
-             struct audio_out_backend* aout)
+             struct audio_out_backend* aout, unsigned int fixed_audio_pos)
 {
     ai->r4300 = r4300;
     ai->ri = ri;
     ai->vi = vi;
     ai->aout = aout;
+    ai->fixed_audio_pos = fixed_audio_pos;
+    ai->audio_pos = 0;
```

Comment:
move ai->audio_pos = 0 in poweron_ai as it is a "state" variable and not a "configure" variable.